src/plugins/intel_cpu/src/emitters/snippets/x64/jit_kernel_emitter.hpp

```diff
@@ -47,39 +85,79 @@ struct jit_snippets_compile_args {
 class jit_kernel_emitter : public jit_container_emitter {
 public:
     jit_kernel_emitter(dnnl::impl::cpu::x64::jit_generator* h, dnnl::impl::cpu::x64::cpu_isa_t isa,
-                       const ov::snippets::lowered::ExpressionPtr& expr);
+                       const ov::snippets::lowered::ExpressionPtr& expr, size_t reg_runtime_params_idx);
```

Comment:
Are you sure we need this complication?
Would it be easier to change the orger of runtime arguments for static kernels:
` callable(&call_args, indexes)`  instead of ` callable(indexes, &call_args)` 
while dynamic impl would be called ` callable(&call_args)`

It's better to align static and dynamic kernels if we can. What do you think?

Reply:
OMG, it's obvious, simpler and better! I don't know why I didn't thank about that 😆 
Thanks a lot!